src/web/app/pages-instructor/view-photo-popover/view-photo-popover.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+/**
+ * Component to display a popover for photos
+ */
+@Component({
+  selector: 'tm-view-photo-popover',
+  templateUrl: './view-photo-popover.component.html',
+  styleUrls: ['./view-photo-popover.component.scss'],
+})
+export class ViewPhotoPopoverComponent implements OnInit {
+
+  @Input()
+  photoUrl: string = '';
+
+  @Input()
+  isContentAvailable: boolean = true;
```

Comment:
What does `isContentAvailable` control?

Reply:
This is for the student list. If there is something between the `tm-view-photo-popover` tag, like the GRQ views, the view photo hyperlink shows in the popover. If its just a opening and closing tag, the view photo hyperlink shows like in student list

Reply:
Then we should have a better naming for it?

Reply:
Alright. How would `isViewPhotoLinkInPopover` sound?

Reply:
That is definitely better than `isContentAvailable` 💯